web/templates/confirm_dialog/confirm_delete_profile_field.hbs

```diff
@@ -1,12 +1,24 @@
 {{#if (eq count 1)}}
     {{#tr}}
         This will delete the <z-profile-field-name></z-profile-field-name> profile field for 1 user.
-        {{#*inline "z-profile-field-name"}}<strong>{{profile_field_name}}</strong>{{/inline}}
+        {{#*inline "z-profile-field-name"}}
+        {{#if rendered_profile_field_name}}
+        <strong class="profile_field_name rendered_markdown">{{rendered_markdown rendered_profile_field_name}}</strong>
+        {{else}}
+        <strong>{{profile_field_name}}</strong>
+        {{/if}}
+        {{/inline}}
     {{/tr}}
 {{else}}
     {{#tr}}
         This will delete the <z-profile-field-name></z-profile-field-name> profile field for <z-count></z-count> users.
         {{#*inline "z-count"}}{{count}}{{/inline}}
-        {{#*inline "z-profile-field-name"}}<strong>{{profile_field_name}}</strong>{{/inline}}
+        {{#*inline "z-profile-field-name"}}
+        {{#if rendered_profile_field_name}}
+        <strong class="profile_field_name rendered_markdown">{{rendered_markdown rendered_profile_field_name}}</strong>
+        {{else}}
```

Comment:
This if-conditions are required now, but am not sure we would need this if we add a migration to set the `rendered_..` fields for existing custom profile fields as well. We have these for stream descriptions as well, but maybe we can remove them as well.

There is no need of any changes for now till we finalize whether we want a migration for existing fields, but just mentioning it here so we do not miss this.